requirements/base.txt

```diff
@@ -244,10 +248,14 @@ pyyaml==5.4.1
     #   apispec
 redis==3.5.3
     # via apache-superset
+requests==2.27.1
```

Comment:
We cannot add this, license is not compatible with Apache

Reply:
I'm not sure when this was changed, but `requests` is now licensed under the APL: https://github.com/psf/requests/commit/22b5a39098223f51fcd2df238e13f9bac86b35a4